data/shubhamku044.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "Shubham Kumar",
+  "bio": "Open Source | Full stack developer",
+  "tags": [
+    "Open Source",
+    "Javascript",
+    "Typescript",
+    "NextJS",
+    "ReactJS",
+    "TailwindCSS"
+  ],
+  "socials": [
+    {
+      "icon": "FaTwitter",
+      "url": "https://twitter.com/shubhamku044"
+    },
+    {
+      "icon": "FaGithub",
+      "url": "https://github.com/shubhamku044"
+    },
+    {
+      "icon": "FaYoutube",
+      "url": "https://www.youtube.com/@shubhamku044"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/shubhamku044/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me here for blogging about devrel and other dev topics",
+      "url": "https://dev.to/shubhamku044",
+      "icon": "FaDev"
+    },
```

[Recommendation] See my feedback on [this line](https://github.com/EddieHubCommunity/LinkFree/pull/7524/files#r1234535017). They both share the same need for improvement.  
```suggestion
      "name": "Follow me on LinkedIn",
      "url": "https://www.linkedin.com/in/shubhamku044/",
      "icon": "FaLinkedin"
    },
    {
      "name": "Follow me here for blogging about devrel and other dev topics",
      "url": "https://dev.to/shubhamku044",
      "icon": "FaDev"
    },
```

---

Please see my comment here https://github.com/EddieHubCommunity/LinkFree/pull/7524/files#r1234671720